### PR TITLE
Fixed Smoke tests from zcs-9234

### PR DESCRIPTION
**Pull request by girishbhamare**
Executed Smoke tests and fixed/skipped follwoing failed tests.

Total Executed: 351
Total Passed: 342
Total Failed: 9
Script Errors: 24

**Failed Tests:**

build/temp/data/soapvalidator/Admin/GAL/GALRequest.xml  :**ZCS-8853 :Regression**

build/temp/data/soapvalidator/Calendar/Resources/CalendarResource.xml: **ZCS-8393**

build/temp/data/soapvalidator/Contacts/AutoComplete/AutoCompleteRequesti18n.xml"**zcs-7999**

build/temp/data/soapvalidator/Contacts/AutoComplete/Ranking/AutoComplete-GAL-Ranking.xml: **ZCS-6163**

build/temp/data/soapvalidator/Contacts/GAL/ZCS-455.xml : **Fixed : script issue: /home/ubuntu/ZCS-455.xml --Removed server="${server.zimbraAdmin}"**

build/temp/data/soapvalidator/Folders/Folder-Retention-Policy.xml : **Fixed Script Issue:Delay after FolderActionRequest**

build/temp/data/soapvalidator/Prefs/Filters/Sieve/Notification.xml : **Need to skip**

build/temp/data/soapvalidator/Search/DelayedIndexing/ZCS-8515-8517.xml-- **Need to add attribute in LC**

### CODE REVIEW

File: data/soapvalidator/Folders/Folder-Retention-Policy.xml [hunk 1]
```diff
@@ -159,7 +159,7 @@
             </t:select>
         </t:response>
     </t:test>
-
+   <t:delay sec="60"/>
```
Comment (jiteshsojitra) [inline data/soapvalidator/Folders/Folder-Retention-Policy.xml:162]: 60s is too much, any other reasonable value?

File: conf/skipped-tests-zimbrax.txt [hunk 2]
```diff
@@ -35,6 +35,15 @@ build/temp/data/soapvalidator/Search/Contacts/Search-Contacts.xml
 build/temp/data/soapvalidator/Search/SearchRequest.xml
 build/temp/data/soapvalidator/Prefs/DataSource/DataSource-Import.xml
 build/temp/data/soapvalidator/Prefs/DataSource/ImapImport/ImapImportBasic.xml
+#ZCS-8853
+build/temp/data/soapvalidator/Admin/GAL/GALRequest.xml
+#ZCS-8393
+build/temp/data/soapvalidator/Calendar/Resources/CalendarResource.xml
+#ZCS-7999
+build/temp/data/soapvalidator/Contacts/AutoComplete/AutoCompleteRequesti18n.xml
+#ZCS--6163
```
Comment (jiteshsojitra) [inline conf/skipped-tests-zimbrax.txt:44]: ZCS-6163

Comment (girishbhamare) [reply conf/skipped-tests-zimbrax.txt:44]: Not sure what are you trying to say

File: data/soapvalidator/Folders/Folder-Retention-Policy.xml (hunk 1 above)
Comment (girishbhamare) [reply data/soapvalidator/Folders/Folder-Retention-Policy.xml:162]: I have tried 30, 40 second as well but its working intermittently.
With 60 sec delay it always paases.

File: conf/skipped-tests-zimbrax.txt (hunk 2 above)
Comment (jiteshsojitra) [reply conf/skipped-tests-zimbrax.txt:44]: #ZCS--6163 to ZCS-6163

Comment (girishbhamare) [reply conf/skipped-tests-zimbrax.txt]: done

File: data/soapvalidator/Folders/Folder-Retention-Policy.xml (hunk 1 above)
Comment (jiteshsojitra) [reply data/soapvalidator/Folders/Folder-Retention-Policy.xml:162]: Hmm.. 60Sec is bad for product if any but its fine according to your analysis.

**Pull request merged**